tests/testnet.spec.js

```diff
@@ -215,7 +215,7 @@ describe('Testing generic actions', () => {
   it(`getFioBalance`, async () => {
     const result = await fioSdk.genericAction('getFioBalance', {})
 
-    expect(result).to.have.all.keys('balance')
+    expect(result).to.have.all.keys('balance','available')
     expect(result.balance).to.be.a('number')
```

Comment:
add `expect(result.available).to.be.a('number')`

Reply:
done